README.md

```diff
@@ -30,68 +30,68 @@ Otherwise, you may use cargo:
 cargo install forc-wallet
 ```
 
-### Initialize
+### Create a wallet
 
-Before creating accounts and signing transactions with them you need to initialize a new HD wallet. To do so:
+Before creating accounts and signing transactions with them you need to create a wallet. To do so:
 
 ```sh
-forc-wallet init
+forc-wallet new
 ```
 
-This will require a password for encyrpting the wallet. After the initialization is done you will be given the mnemonic phrase of the wallet.
+This will require a password for encrypting the wallet. After the wallet is created you will be shown the mnemonic phrase.
+
+> Note: You will need your password for signing and account derivation, and you will need your mnemonic phrase if you wish to recover your wallet in the future.
+
+### Import a wallet
 
-You can also initialize a wallet with your existing mnemonic phrase by using `forc-wallet import`.
+To import a wallet from an existing mnemonic phrase, use:
+
+```sh
+forc-wallet import
+```
 
 > Note: `forc-wallet` adheres to the [Web3 Secret Storage Definition](https://ethereum.org/en/developers/docs/data-structures-and-encoding/web3-secret-storage) and accepts paths to wallet files that adhere to this standard.
 
 ### Create an account
 
-To create an account for the initialized wallet, you can run:
+To create an account for the wallet, you can run:
 
 ```sh
-forc-wallet new
+forc-wallet account new
 ```
 
-This will require your wallet password (the one that you choosed in the initialization step). This will always derive the next account.
-
-### Sign a transaction
+This will require your wallet password (the one that you chose during creation). This command will always derive the next account that has not yet been derived locally.
 
-To sign a transaction, you need to have the transaction ID. You can generate a transaction and get its ID using `forc-client`. Signing the transaction once you have the ID is simple:
+To list all accounts derived so far, use the following:
 
 ```sh
-forc-wallet sign --id <transaction_id> --account-index <account_index>
+forc-wallet accounts
```

Comment:
Mminor nit: There was some discussion around users not being sure if they should be calling `forc-wallet` or `forc wallet`; just double-checking that this update to the readme is going with the preferred method.

Reply:
Yeah we prefer using `forc wallet` so that `forc` manages the execution of the binary. This is kind of confusing but we might have a motivation to keep this like `forc-wallet` as well specifically for this repo's readme:

Since `forc-wallet` is a separate binary that does not require any functionality of `forc` it is possible to only install `forc-wallet` even if you don't have `forc` installed. So for the readme of this repo we might use `forc-wallet` but in the quick start tutorials and `forc` documentation we should definitely use `forc wallet`. 

I am good with either changing it or keeping it or maybe having a small note that states that if `forc` is installed `forc wallet` is usable and preferred.

Reply:
Yeah I'll address this in a follow-up PR, with a `> Note:` toward the top of the README that mentions something like "if you have `forc` installed, you can use this plugin via `forc wallet`" or something along these lines.